### ISSUE TITLE
release issues for 1.1-9

### ISSUE THREAD

**Issue by bbolker** [lme4/lme4#324]
We should release version 1.1-9 soon (CRAN-compatible imports, etc. etc.)

Release-critical in my opinion:

* Finish the thread that Martin's improvements to `glmer.nb` started (#319, #322)
* Check downstream packages (I can do that; need to update my scripts a little bit)
* we seem to have some [test failures on Windows](https://www.r-project.org/nosvn/R.check/r-devel-windows-ix86+x86_64/lme4-00check.html) to fix (probably just a matter of tweaking tolerances?)

**Comment by mmaechler** [comment]
Looking at "3" - test failures on Windows: the first of the two is clearly ok by losing tolerance... I've also done something with the 2nd (more messages)... but am much less sure.
Can you upload to Winbuilder and forward us the link?

**Comment by bbolker** [comment]
I think everyone will be getting the win-builder e-mails since the maintainer address is (still) `bbolker+lme4@gmail.com` (which forwards to the maintainer list).  First attempt failed on some glitches, trying again ...

**Comment by bbolker** [comment]
OK, now fails for exactly the same reason as #319 (so those two issues are essentially merged):

```
** running tests for arch 'x64' ... ERROR
Running the tests in 'tests/AAAtest-all.R' failed.
Last 13 lines of output:
  1. Failure (at test-glmernb.R#16): basic ---------------------------------------
  fixef(m.nb) not equal to coef(m.glm)
  6/6 mismatches (average diff: 1.03).
  First 6:
   pos      x         y   diff
     1  1.650  7.76e-07  1.650
     2  0.767 -1.09e-06  0.767
     3  1.011 -2.74e-06  1.011
     4  1.512  5.04e-07  1.512
     5 -0.615  1.11e-06 -0.615
     6 -0.610  8.04e-07 -0.610
  Error: Test failures
```

**Comment by bbolker** [comment]
since we've just released 1.1-10 I think it's OK to close this.  The `glmer.nb` issues are still there, but are represented by other open issues ...
